RecursiveExtractor/ExtractorOptions.cs

```diff
@@ -49,5 +51,10 @@ public class ExtractorOptions
         /// Run in parallel when possible
         /// </summary>
         public bool Parallel { get; set; }
```

Comment:
Why not set default here to true?

Reply:
Because parallelizing makes it not a true enumerable since it does parallel calculations and builds a queue before returning off that queue.